deployment/ansible/deploy.yml

```diff
@@ -73,6 +73,22 @@
 
   tasks:
     - block:
+      - name: Schedule downtime
+        run_once: yes
+        uri:
+          url: "{{ datadog_api_url }}"
+          method: POST
+          status_code: 200
+          timeout: 10
+          body_format: json
+          body:
+            scope: "env:{{ env }}"
+            message: "Deploy"
+            end: "{{ lookup('pipe', 'date -v+{{ downtime }} +%s') }}"
```

Comment:
I see the Circle CI config [uses Ubuntu 16.04](https://github.com/aeternity/docker-builder/blob/a143797b693c31ba928b4c60390859e33b151225/Dockerfile#L1). I see option `-v` on macOS though not on Ubuntu manpage ([coreutils](http://manpages.ubuntu.com/manpages/xenial/en/man1/date.1.html)). Could you please point me to where this is defined?

Reply:
Oops, I tested it on OSX, I'll take a deeper look.

Reply:
@lucafavatella fixed by 1a9d837d